cmd/iaviewer/main.go

```
@@ -25,50 +25,257 @@ const (
+		if err := tree.DeleteVersionsTo(int64(targetVersion)); err != nil {
+			fmt.Fprintf(os.Stderr, "Error delete version: %s\n", err)
+		}
+	case "delete-tree":
+		dir, prefix := args[1], args[2]
+
+		db, err := OpenGoLevelDB(dir)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Error reading data: %s\n", err)
+		}
+
+		deleteKeysWithPrefix(prefix, db)
```

golangci-lint (1.64.8)

Error return value is not checked

(errcheck)